README.md

```diff
@@ -391,6 +391,14 @@ No additional parameters required
 
 `GET /api/tags`
 
+### Running the Authors Haven
 
+`python3 manage.py runserver --settings=authors.settings.dev`
 
+### Running tests on Authors Haven
 
+`python3 manage.py test --settings=authors.settings.test`
+
+### Live Application URL
```

Comment:
Since these are different ways of running the application
shouldn't we can it 
**running the application on heroku**